images/proxysql/tests/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "version" {
+  digest = var.digest
+  script = "${path.module}/01-version.sh"
+}
```

Comment:
I think we can migrate the helm test in this PR. There are some examples in the repo now, for example https://github.com/chainguard-images/images/blob/main/images/stakater-reloader/tests/main.tf#L21

Reply:
It does make sense! I didn't notice this. So how can we ensure pods are up and running? We used to use `kubectl wait ...`, in this case, maybe I can use [wait](https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#wait). 🤔

Reply:
Migrated to TF! Could you please check? @imjasonh